formats/log/README.md

```diff
@@ -45,25 +45,42 @@ signatures, the format itself is not restricted to this scheme.
 The checkpoint body is of the form:
 
 ```text
-<Ecosystem & version string>
+<Origin string>
 <Decimal log size>
 <Base64 log root hash>
 [otherdata]
 ```
 
 The first 3 lines of the body **MUST** be present in all Checkpoints.
-`otherdata` is opaque and optional, and, if necessary, can be used to tie extra
-data to the checkpoint, however its format must conform to the sumdb signed
-note spec (e.g. it must not contain blank lines.)
+
+* `<Origin string>` should be a unique identifier for the log identity which issued the checkpoint.
+  The exact format of this is left undefined, but examples of types of value to put here
+  are the log's URL, a base64 encoded hash of its public key, etc.
+
+  The presence of this identifier forms part of the log claim, and prevents witness
+  signatures being moved between log checkpoints with identical contents, preventing a
+  particular type of targetted attack.
```

Comment:
I think this is just spelled "targeted" but maybe that's an American thing?

Reply:
fixxed